claims_db/phclaims/stage/tables/create_stage_mcare_mbsf_ab.yaml

```diff
@@ -0,0 +1,56 @@
+schema: stage
+table: mcare_mbsf_ab
+years:
+vars:
+    bene_id: VARCHAR(15) collate SQL_Latin1_General_Cp1_CS_AS
```

Comment:
I assume you're not renaming this to id_mcare until a later point?

Reply:
I harmonized the 2011-2014 (MBSF AB) and 2015-2016 (MBSF ABCD) data, but tried to retain as many of the original column names as possible so that we can still use data dictionaries provided by RESDAC. The elig and timevar tables will use id_mcare.